app/controllers/projects_controller.rb

```diff
@@ -254,6 +255,7 @@ def set_project
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def project_params
+    params[:name] = sanitize(params[:name])
```

Comment:
:name is nested right?

Reply:
yes. thanks

Reply:
moved sanitize to after the require/permit, at which point name is no longer nested